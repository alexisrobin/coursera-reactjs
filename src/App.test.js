import { expect } from 'chai'
import React from 'react'
import { mount, render, shallow } from 'enzyme'
import Main from './components/MainComponent';
import App from './App'

describe('<App />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<App />)

    expect(wrapper.find(Main)).to.have.length(1)
  })

  it('should match its reference snapshot', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).to.matchSnapshot()
  })
})