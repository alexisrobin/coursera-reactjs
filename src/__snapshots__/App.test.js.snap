// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match its reference snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <div>
      <withRouter(Connect(Main)) />
    </div>
  </BrowserRouter>
</Provider>
`;
